# HG changeset patch
# User Sergi Almacellas Abellana <sergi@koolpi.com>
Add company access rule for inventories

issue4482
review13891002

Index: trytond/trytond/modules/stock/inventory.xml
===================================================================

--- a/trytond/trytond/modules/stock/inventory.xml
+++ b/trytond/trytond/modules/stock/inventory.xml
@@ -57,6 +57,15 @@
             <field name="name">inventory_line_tree</field>
         </record>

+        <record model="ir.rule.group" id="rule_group_inventory">
+            <field name="model" search="[('model', '=', 'stock.inventory')]"/>
+            <field name="global_p" eval="True"/>
+        </record>
+        <record model="ir.rule" id="rule_inventory">
+            <field name="domain" eval="[('company', 'in', Eval('user', {}).get('companies', []))]" pyson="1"/>
+            <field name="rule_group" ref="rule_group_inventory"/>
+        </record>
+
         <record model="ir.model.access" id="access_inventory">
             <field name="model" search="[('model', '=', 'stock.inventory')]"/>
             <field name="perm_read" eval="False"/>

